### ISSUE TITLE
GraphicsCriticalError: |[0][GFX1-]: glxtest: libpci missing 

### ISSUE THREAD

**Issue by sibalzer** [sibalzer/primelooter#11]
weird error

```log
2021-09-24 01:19:31 [WARNING] Loop <_UnixSelectorEventLoop running=False closed=True debug=False> that handles pid 12136 is closed
primelooter    | 2021-09-25 01:20:37 [ERROR] Error Timeout 30000ms exceeded.
primelooter    | =========================== logs ===========================
primelooter    | <launching> /ms-playwright/firefox-1281/firefox/firefox -no-remote -headless -profile /tmp/playwright_firefoxdev_profile-LfYaYc -juggler-pipe -silent
primelooter    | <launched> pid=13084
primelooter    | [pid=13084][err] *** You are running in headless mode.
primelooter    | [pid=13084][out] Crash Annotation GraphicsCriticalError: |[0][GFX1-]: glxtest: libpci missing (t=20.1245) [GFX1-]: glxtest: libpci missing
primelooter    | [pid=13084][out] Crash Annotation GraphicsCriticalError: |[0][GFX1-]: glxtest: libpci missing (t=20.1245) |[1][GFX1-]: glxtest: Unable to open a connection to the X server (t=20.1656) [GFX1-]: glxtest: Unable to open a connection to the X server
primelooter    | [pid=13084][out] Crash Annotation GraphicsCriticalError: |[0][GFX1-]: glxtest: libpci missing (t=20.1245) |[1][GFX1-]: glxtest: Unable to open a connection to the X server (t=20.1656) |[2][GFX1-]: glxtest: libEGL initialize failed (t=20.1656) [GFX1-]: glxtest: libEGL initialize failed
primelooter    | [pid=13084][out] Crash Annotation GraphicsCriticalError: |[0][GFX1-]: glxtest: libpci missing (t=20.1245) |[1][GFX1-]: glxtest: Unable to open a connection to the X server (t=20.1656) |[2][GFX1-]: glxtest: libEGL initialize failed (t=20.1656) |[3][GFX1-]: No GPUs detected via PCI (t=20.1861) [GFX1-]: No GPUs detected via PCI
primelooter    | ============================================================
primelooter    | 2021-09-25 01:21:38 [ERROR] Error It looks like you are using Playwright Sync API inside the asyncio loop.
primelooter    | Please use the Async API instead.
```

**Comment by sibalzer** [comment]
maybe b4d7035 also fixes #7

**Comment by sibalzer** [comment]
seems to be fixed.
